projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/components/RunJob.tsx

```diff
@@ -49,7 +51,7 @@ export default class RunJobDialog extends Component<IRunJobDialogProps> {
         if(!value){
             value = this.props.jobPath;
         }
-        sessionStorage.setItem("job-path", value)
+        jobData.set(JobProperties.path, value);
```

Comment:
is the stateful struct a vdkGlobalStorage, a custom alternative to Jupyter's sessionStorage? could you describe why we have it, in the struct code documentation?

Reply:
done